client/signup/steps/user/index.jsx

```diff
@@ -259,7 +260,23 @@ export class UserStep extends Component {
 		}
 
 		if ( isReskinned && 0 === positionInFlow ) {
-			subHeaderText = '';
+			if ( this.props.isSocialFirst ) {
+				subHeaderText = '';
+			} else {
+				const { queryObject } = this.props;
+				if ( queryObject?.variationName && isNewsletterFlow( queryObject.variationName ) ) {
+					subHeaderText = translate( 'Already have a WordPress.com account? {{a}}Log in{{/a}}', {
+						components: { a: <a href={ loginUrl } rel="noopener noreferrer" /> },
+					} );
+				} else {
+					subHeaderText = translate(
+						'First, create your WordPress.com account. Have an account? {{a}}Log in{{/a}}',
+						{
+							components: { a: <a href={ loginUrl } rel="noopener noreferrer" /> },
+						}
+					);
+				}
+			}
```

Comment:
Would appreciate a line about this change.

Reply:
Code starting from `const { queryObject } = this.props;` is what was already there in the User step. 
`subHeaderText = '';` was added when making this a Social First signup. Now I'm reintroducing the old subheader when this is the old User step